salt/modules/zypperpkg.py

```diff
@@ -983,7 +985,11 @@ def _is_match(pkgname):
         return False
 
     root = kwargs.get('root') or None
-    for node in __zypper__(root=root).xml.call('se', '-s', *targets).getElementsByTagName('solvable'):
+    for node in (
+        __zypper__(root=root)
+        .xml.call("se", "-s", *targets, success_retcodes=[104])
```

Comment:
Would be awesome not to have a magic hardcodee but define it somewhere as a constant above.

Reply:
Good idea, yes.